docs/api/toggle.md

```diff
@@ -24,183 +17,58 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
-}
-```
-
-
-### Customizing Width
-
-Adjusting the width of the toggle **smaller** will result in a narrower track, with the handle remaining the default width. If desired, set `--handle-width` to make the handle narrower.
-
-```css
-ion-toggle {
-  width: 40px;
-}
-```
-
-Adjusting the width of the toggle **larger** will result in a wider track, with the handle remaining the default width. If desired, set `--handle-width` to make the handle wider.
-
-```css
-ion-toggle {
-  width: 100px;
-}
-```
-
-### Customizing Height
 
-Adjusting the height of the toggle **smaller** than the default will result in the handle height auto-sizing itself to the track. In `ios` the handle is shorter than the track (`100% - 4px`) and in `md` the handle is taller than the track (`100% + 6px`).
+Toggles are switches that change the state of a single option. They can be switched on or off by pressing or swiping them. Toggles can also be checked programmatically by setting the `checked` property.
 
-```css
-ion-toggle {
-  height: 15px;
-}
-```
-
-:::note
- Note: this does not affect the handle width, width should be set using `--handle-width`.
-:::
-
-
-Adjusting the height of the toggle **larger** will keep the handle in the center at the default height. This can be modified by setting `--handle-height` which can be set to any amount but will not exceed the `--handle-max-height`.
-
-```css
-ion-toggle {
-  height: 60px;
-}
-```
-
-:::note
- Note: this does not affect the handle width, width should be set using `--handle-width`.
-:::
+## Basic Usage
 
+import Basic from '@site/static/usage/toggle/basic/index.md';
 
-### Customizing Spacing
+<Basic />
 
-The spacing refers to the horizontal gap between the handle and the track. By default, the handle has `2px` of spacing around it in `ios` **only**. In `md` mode there is no default spacing.
 
-To remove the **horizontal** spacing, set `--handle-spacing` to `0px`.
+## On / Off Labels
 
-```css
-ion-toggle {
-  --handle-spacing: 0px;
-}
-```
+Toggles can enable on/off labels by setting the `enableOnOffLabels` property. This is important for accessibility as it makes it easier to differentiate between a checked and unchecked toggle.
 
-Due to the handle having a fixed height, to remove the spacing on the top and bottom, set the height to 100%.
+import OnOff from '@site/static/usage/toggle/on-off/index.md';
 
-```css
-ion-toggle {
-  --handle-spacing: 0px;
-  --handle-height: 100%;
-}
-```
+<OnOff />
 
 
-### Customizing Border Radius
+## List Toggles
 
-The `--handle-border-radius` can be used to change the `border-radius` on the handle.
+Toggles can be slotted inside of an item in a list. Native designs place toggles at the end of an item, but they can also be placed at the start.
 
-```css
-ion-toggle {
-  --handle-border-radius: 14px 4px 4px 14px;
-}
-```
+import List from '@site/static/usage/toggle/list/index.md';
 
-To target the `border-radius` only when the toggle is checked, target `.toggle-checked`:
+<List />
```

Comment:
I think we should keep something showing toggles in a list only because that is going to be the number 1 recommended use case for them. iOS doesn't even recommend using toggle outside of a list:

![Screen Shot 2022-09-28 at 1 25 49 PM](https://user-images.githubusercontent.com/6577830/192848831-7c0ff368-79be-44d3-8524-bb40669bcb4a.png)

And Material Design only shows them in lists:
 
![Screen Shot 2022-09-28 at 1 26 32 PM](https://user-images.githubusercontent.com/6577830/192849049-563bb010-1aae-430d-a59b-0eb22172ea15.png)

The [list documentation](https://ionicframework.com/docs/api/list) doesn't actually show any toggles in the examples, so I think it's good to have something for people to easily see and copy/paste.

Reply:
In that case, I would argue the basic usage playground should show them in a list, rather than having a separate example just for that.

Reply:
Yeah I wanted to show normal and disabled in basic but I can just combine that into a list. 👍